src/Peering/Peering.Test/ScenarioTests/PeeringCommon.ps1

```diff
@@ -75,15 +75,43 @@ function maxAdvertisedIpv6
 function getHash
 {
 $hash = getPeeringVariable "Hash" $ipGenerator.BuildHash()
+Write-Debug("The hash $hash");
```

Comment:
@rileymckenna in this file, it should be `Write-Debug "Message"` instead of `Write-Debug("Message")`